common/http.hpp

```diff
@@ -285,9 +285,8 @@ class ClientConfig {
 	ClientConfig(string server_cert_path);
 	~ClientConfig();
 
-#ifdef MENDER_USE_BOOST_BEAST
-	ssl::context ctx_ {ssl::context::tls_client};
-#endif // MENDER_USE_BOOST_BEAST
+public:
+	string server_cert_path;
```

Comment:
Time to turn `ClientConfig` into a `struct`? I mean, it should, by definition be a passive entity.

Reply:
Indeed, maybe it wouldn't have even happened then. Fixed.